packages/atomic/cypress/integration/facet-utils.ts

```diff
@@ -58,3 +84,112 @@ export async function assertSortCriteria(
 
   expect(firstRequestBodyFacets).to.have.property('sortCriteria', sortOption);
 }
+
+export function convertFacetValueToAPIformat(
+  value: string,
+  facetTypeOption?: string,
+  valueSeparator?: string
+) {
+  value = value.trim();
+  valueSeparator = valueSeparator ? valueSeparator : 'to';
+  const splitByValue = ` ${valueSeparator} `;
+  const start = value.split(splitByValue)[0];
+  const end = value.split(splitByValue)[1];
+
+  switch (facetTypeOption) {
+    case FacetType.numericFacet: {
+      const formtedStart = revertFormatNumericFacet(start);
+      const formatedEnd = revertFormatNumericFacet(end);
+      return `${formtedStart}..${formatedEnd}`;
+    }
+    case FacetType.dateFacet: {
+      const formtedStart = `${revertFormatedDateFacet(start)}@00:00:00`;
+      const formatedEnd = `${revertFormatedDateFacet(end)}@00:00:00`;
+      return `${formtedStart}..${formatedEnd}`;
+    }
+    default:
+      return value;
+  }
+}
+
+export interface NumericRange {
+  start: number;
+  end: number;
+}
+export interface DateRange {
+  start: string;
+  end: string;
+}
```

Comment:
Following on the earlier comments, you could use these interfaces as the types of the arrays: `NumericRange[]`, `DateRange[]`.